src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,169 @@
+                    className={cs('', {
+                      'has-background-warning':
+                        location.pathname.slice(8) === person.slug,
+                    })}
+                    data-cy="person"
+                  >
+                    <td>
+                      <Link
+                        className={cs('', {
+                          'has-text-danger': person.sex === 'f',
+                        })}
+                        to={`/people/${person.slug}`}
+                        onClick={event => {
+                          event.preventDefault();
+
+                          navigate(`/people/${person.slug}`);
+                        }}
+                      >
+                        {person.name}
+                      </Link>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+                    <td>
+                      {!!person.motherName ? (
+                        !!findPerson(person.motherName) ? (
+                          <Link
+                            className="has-text-danger"
+                            to={`/people/${findPerson(person.motherName)?.slug}`}
+                            onClick={event => {
+                              event.preventDefault();
+
+                              if (person.motherName) {
+                                navigate(
+                                  `/people/${findPerson(person.motherName)?.slug}`,
+                                );
+                              }
+                            }}
+                          >
+                            {person.motherName}
+                          </Link>
+                        ) : (
+                          person.motherName
+                        )
+                      ) : (
+                        '-'
+                      )}
+                    </td>
+                    <td>
+                      {!!person.fatherName ? (
+                        !!findPerson(person.fatherName) ? (
+                          <Link
+                            to={`/people/${findPerson(person.fatherName)?.slug}`}
+                            onClick={event => {
+                              event.preventDefault();
+
+                              if (person.fatherName) {
+                                navigate(
+                                  `/people/${findPerson(person.fatherName)?.slug}`,
+                                );
+                              }
+                            }}
+                          >
+                            {person.fatherName}
+                          </Link>
+                        ) : (
+                          person.fatherName
+                        )
+                      ) : (
+                        '-'
+                      )}
+                    </td>
+                  </tr>
+                ))}
```

Comment:
logic for mother and father is the same and can be move in separate component like PersonLink